NDB.Covid19/NDB.Covid19/Models/DTOsForServer/DailyNumbersDTO.cs

```diff
@@ -3,7 +3,7 @@ namespace NDB.Covid19.Models.DTOsForServer
 {
     public class DailyNumbersDTO
     {
-        public FHIStatisticsDTO FHIStatistics { get; set; }
-        public AppStatisticsDTO AppStatistics { get; set; }
+        public FHIStatisticsDTO SSIStatistics { get; set; }
```

Comment:
Rename SSIStatistics

Reply:
At backend as well, if this is the reason

Reply:
Yes, it will be renamed in the backend later, together with adding some missing numbers, and then updated in the app code